lib/schema/deviceMeta.js

```diff
@@ -30,17 +30,28 @@ var UNKNOWN_PREV = 'status/unknown-previous';
 var idFields = ['type', 'subType', 'time', 'deviceId'];
 schema.registerIdFields('deviceMeta', idFields);
 
-var suspendedReasons = {
-  'manual': true,
-  'low_glucose': true,
-  'alarm': true
-};
-
-var resumedReasons = {
+var statusReasons = {
```

Comment:
I never noticed this before, but it's a strange construct to me, especially when the same items are repeated later in the code, and also when Object.keys() is used on it as well. 

Can we unify this, please? I'd suggest that statusReasons becomes an array passed to schema.in, and that _.contains() gets used for checking for membership.